### DISCUSSION TITLE
Does Phaser Editor support 32 bits?

### DISCUSSION THREAD

**Discussion by RedEdge967** [PhaserEditor2D/PhaserEditor2D-v3#222] [Q&A] [answered]
Hi there!

I just checked this and wanted to make a game using this. I wanted to know if this editor supports `windows 7 32 bit?`

Thank you!

**Comment by PhaserEditor2D** [comment, marked as answer]
Hi. No. It doesn't support 32 bits.

**Reply by RedEdge967** [reply]
thank you! btw your work is good

**Reply by PhaserEditor2D** [reply]
Thanks!
